integration-tests/smallrye-jwt-token-propagation/src/main/resources/application.properties

```diff
@@ -12,4 +12,4 @@ smallrye.jwt.new-token.override-matching-claims=true
 
 quarkus.http.auth.proactive=false
 
-quarkus.native.additional-build-args=-H:IncludeResources=.*\\.pem
+quarkus.native.additional-build-args=-H:IncludeResources=publicKey.pem
```

Comment:
This update ensure that a signing `privateKey.pem` mentioned above with `smallrye.jwt.sign.key.location=/privatekey.pem` is not included manually.

In this test, `publicKey.pem` has to be added manually because it is loaded manually in the test code, it is not part of the smallrye-jwt-* config